routers/api/v1/shared/runners.go

```diff
@@ -30,3 +34,56 @@ func GetRegistrationToken(ctx *context.APIContext, ownerID, repoID int64) {
 
 	ctx.JSON(http.StatusOK, RegistrationToken{Token: token.Token})
 }
+
+func ListRunners(ctx *context.APIContext, ownerID, repoID int64) {
+	runners, total, err := db.FindAndCount[actions_model.ActionRunner](ctx, &actions_model.FindRunnerOptions{
+		OwnerID:     ownerID,
+		RepoID:      repoID,
+		ListOptions: utils.GetListOptions(ctx),
+	})
```

Comment:
It seems not right due to the framework's buggy behavior. A zero field won't appear in the "WHERE" part.